modules/protect/math-fallback.php

```diff
@@ -124,8 +124,16 @@ static function math_form() {
 			?>
 			<div style="margin: 5px 0 20px;">
 				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
+				<label for="sum">
+					<?php echo $num1;
+					// the use of "left: -999em; position: absolute;" below is for users with assistive technology since "display: none" hides from all users. ?>
+					<span style="left: -999em; position: absolute;">plus</span>
```

Comment:
Since it's a word expected to be read by screen readers, it should be made available for translation.